exercises/02-publish-subscribe/after/Divergent.Customers/Config/EndpointConfig.cs

```diff
@@ -33,8 +34,13 @@ public void Customize(EndpointConfiguration endpointConfiguration)
             endpointConfiguration.UseSerialization<JsonSerializer>();
             endpointConfiguration.Recoverability().Delayed(c=>c.NumberOfRetries(0));
             endpointConfiguration.UseContainer<AutofacBuilder>(c => c.ExistingLifetimeScope(container));
-            endpointConfiguration.UseTransport<MsmqTransport>()
-                .ConnectionString("deadLetter=false;journal=false");
+
+            var routing = endpointConfiguration.UseTransport<MsmqTransport>()
+                .ConnectionString("deadLetter=false;journal=false")
+                .Routing();
+
+            routing.RegisterPublisher(typeof(OrderSubmittedEvent), "Divergent.Sales");
```

Comment:
How well did you test this everything? In XML config it included the entire assembly, now just a single message?